### PR TITLE
Added circular Keyboard to ucrCalculator

### PR DESCRIPTION
**Pull request by Wycklife**
Fixes the part 2 of #5332 

In progress

### CODE REVIEW

**Comment by Wycklife** [comment]
This has some minor mistake and I'm opening a new pull request

**Comment by shadrackkibet** [comment]
I see conflicts here, could you please sort this out.

File: instat/ucrCalculator.vb [hunk 1]
```diff
@@ -1703,9 +1703,38 @@ Public Class ucrCalculator
     End Sub
     Private Sub cmdCircMedian_Click(sender As Object, e As EventArgs) Handles cmdCircMedian.Click
         If chkShowArguments.Checked Then
-            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("circular::mean.circular(x = )", 2)
+            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("circular::median.circular(x = )", 2)
         Else
-            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("circular::mean.circular()", 1)
+            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("circular::median.circular()", 1)
+        End If
+    End Sub
```
Comment (shadrackkibet) [inline instat/ucrCalculator.vb]: you need a space after every sub.

File: instat/ucrCalculator.vb [hunk 2]
```diff
@@ -1694,4 +1694,47 @@ Public Class ucrCalculator
...
+    Private Sub cmdMedianHL_Click(sender As Object, e As EventArgs) Handles cmdMedianHL.Click
+        If chkShowArguments.Checked Then
+            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("circular::medianHL.circular(x = )", 2)
+        Else
+            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("circular::MedianHL.circular()", 1)
+        End If
+    End Sub
+    Private Sub cmdCircVar_Click(sender As Object, e As EventArgs) Handles cmdCircVar.Click
+        If chkShowArguments.Checked Then
+            ucrReceiverForCalculation.AddToReceiverAtCursorPosition("circular::var.circular(x = )", 2)
```
Comment (shadrackkibet) [inline instat/ucrCalculator.vb]: Make sure you add all the optional parameters here. This applies to other options as well.

**Comment by Wycklife** [comment]
I am closing this pull request because the branch has conflict . I'm creating a new pull request with a new branch.

**Pull request closed without merging**